libraries/matrix/test/src/main/kotlin/io/element/android/libraries/matrix/test/verification/FakeSessionVerificationService.kt

```diff
@@ -20,17 +20,22 @@ import io.element.android.libraries.matrix.api.verification.SessionVerificationD
 import io.element.android.libraries.matrix.api.verification.SessionVerificationService
 import io.element.android.libraries.matrix.api.verification.SessionVerifiedStatus
 import io.element.android.libraries.matrix.api.verification.VerificationFlowState
+import io.element.android.tests.testutils.lambda.LambdaNoParamRecorder
+import io.element.android.tests.testutils.lambda.lambdaRecorder
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 
-class FakeSessionVerificationService : SessionVerificationService {
+class FakeSessionVerificationService(
+    var skipVerificationResult: LambdaNoParamRecorder<Boolean> = lambdaRecorder<Boolean>(ensureNeverCalled = false, block = { false })
```

Comment:
`ensureNeverCalled` is set to false by default, so `ensureNeverCalled = false` could be removed, but maybe to set the default value of `skipVerificationResult` it should be set to `true`, to actually correctly cover other unrelated tests?
```suggestion
    var skipVerificationResult: LambdaNoParamRecorder<Boolean> = lambdaRecorder<Boolean>(ensureNeverCalled = true, block = { false })
```

No sure about it though, but this is what I would do.